File: WebContent/extras/c#-processing/FigureSet.cs

using System.Collections.Generic;

namespace shape_detect
{
    public class FigureSet
    {
        private List<Figure> figures;

        public int Size
        {
            get { return figures.Count; }
        }

        public Figure this[int index]
        {
            get { return figures[index]; }
        }

        public FigureSet()
        {
            figures = new List<Figure>();
        }

        public void AddFigure(Figure figure)
        {
            figures.Add(figure);
        }
    }
}

File: src/com/scrbl/model/FigureSet.java
package com.scrbl.model;

public class FigureSet
{
	private java.util.ArrayList<Figure> figures;

	public final int getSize()
	{
		return figures.size();
	}

	public final Figure getItem(int index)
	{
		return figures.get(index);
	}

	public FigureSet()
	{
		figures = new java.util.ArrayList<Figure>();
	}

	public final void AddFigure(Figure figure)
	{
		figures.add(figure);
	}
}